_posts/2018-11-21-how-we-work-exercise-and-stand-desk.md

```diff
@@ -0,0 +1,61 @@
+# How We Work: Exercise and Stand Desks
+
+by Cameron Gera (@gera-cameron)
+
+[PHOTO HERE]
+
+## Why
+
+The culture of our engineering team here at ITProTV encourages healthy habits. We all enjoy taking
+care of our bodies, so why wouldn't we want to do the same at work! The two main ways we do this is
+through, you guessed it, team exercise and stand desks. There is a lot of research out there on what
+improves work place morale, but for us these two elements help our team be the happiest and most
+productive we can be.
+
+## Team Exercise
+
+This element of our work day occurs at the 50th of every hour between eight and five. We have a
```

Comment:
You could remove the sentence starting with 'We have a Slack reminder' if you're looking for stuff to cut.